### ISSUE TITLE
Only geometry node editors are covered in the reference manual, but what about the other node types? 

### ISSUE THREAD

**Issue by raindropsfromsky** [Bforartists/Manual#515]
I am trying to learn node-based design, so I decided to read up the reference manual on that topic.

I noticed that the reference manual describes only one type of node editor: "geometry node".

So I was curious whether there are other types of nodes. When I checked with ChatGPT, it gave the following response:

> There are 4 main types of nodes in Blender:
> 
> - Shader Nodes: These nodes are used to create and edit materials. They can be used to control the color, reflectivity, roughness, and other properties of a material.
> - Texture Nodes: These nodes are used to create and apply textures to objects. They can be used to add things like images, patterns, and noise to materials.
> - Color Nodes: These nodes are used to manipulate colors. They can be used to mix colors, change their brightness and contrast, and convert them between different color spaces.
> - Vector Nodes: These nodes are used to manipulate vectors. They can be used to add, subtract, multiply, and divide vectors, as well as to transform them between different coordinate spaces.
> 
> In addition to these four main types of nodes, there are also a number of other types of nodes available, such as:
> 
> - Converter Nodes: These nodes are used to convert data between different formats. For example, a Color Converter node can be used to convert a color from RGBA to HSV.
> - Script Node: This node allows you to run Python scripts inside Blender. This can be used to create custom nodes or to automate tasks.
> - Node Groups: These allow you to group nodes together and treat them as a single node. This can be useful for organizing complex node setups or for creating reusable node templates.
> - The "Geometric" nodes are a newer type of node that was introduced in Blender 2.93. They are used to procedurally generate and modify geometry. They can be used to create things like meshes, curves, and particles.

Well, the reference manual seems to explain only the last category in this whole list!

The manual should cover these nodes as well.

**Comment by ReinerBforartists** [comment]
I've moved the issue to the manual tracker.

**Comment by raindropsfromsky** [comment]
Thanks for the information.

Just like the actual product, a manual also has usability issues.
From a learner's perspective, I'd like to highlight a few usability issues for the manual.

Blender is a complex app. A learner would not realize the workings of an interface just by finding out what each control does individually. He needs to know the overall function, how the controls interact, and also if there are any interdependency of controls. He would also benefit by learning about some patterns.

Therefore, the manual should include these inputs. It would be great if icons are used to set this text apart. (Many books use icons for insets that provide tips, cautions, examples, cross-references, etc.)

1. All the nodes are similar in nature. So there should be a common explanation about how it is represented, how the dots are color-coded (what different colors mean), how they behave, how to interconnect them, how to break the connection, how to insert a node, how to find a node that will serve the desired effect, etc. <br><br>All chapters on the various types of nodes should cross-refer to this common explanation. <br><br>IMHO, the best place for this is in Chapter#3 (Interface). <br><br>All types of nodes should be summed up here, and cross-links to different chapters should be provided from here.<br><br>
2. In every chapter that has text about nodes, there should be "see also-" links to the other types of nodes, and also to the explanatory common text.

Thanks!

**Comment by ReinerBforartists** [comment]
>I did not find such answers in the manual.

I think we have discussed this isuse before. The reference manual is no tutorial. That's not the job. The reference manual explains the tools. This can also include a short tutorial how to use the tool. But this must happen based on the UI location. Nodes is a very special chapter in itself since it is some kind of programming. And teaching programming does simply not belong into the reference manual.

The Blender manual does it actually like you suggest. This is what makes it so hard to follow it. With the price that as a reference manual the Blender manual fails miserably. It is plain unsearchable, crowded with informations and even plain opinions that doesn't belong into it. And the actual information is burried and hard to find. In case it even exists.

But thanks for your suggestions. Let's see what @Draise14 means here. I have nothing to say against a tutorial chapter about how to use nodes.

**Comment by ReinerBforartists** [comment]
I just start to realize that the chapter that you are asking for already exists. General functionality is always discussed at the very beginning of the editors.

10 Editors - Compositor.odt
12 - Geometry Node Editor.odt
13 Editors - Shader Editor.odt

11 Editors - Texture Node Editor.odt - this editor is as told not documented since it is dysfunctional.

**Comment by ReinerBforartists** [comment]
Closing the issue. Many thanks for reporting.
